src/cpp/_pythia8.cpp

```diff
@@ -257,6 +331,16 @@ PYBIND11_MODULE(_pythia8, m)
         .def("children", event_array_children)
         .def("reset", &Event::reset)
         .def("append", py::overload_cast<int, int, int, int, double, double, double, double, double, double, double>(&Event::append), "pdgid"_a, "status"_a, "col"_a, "acol"_a, "px"_a, "py"_a, "pz"_a, "e"_a, "m"_a = 0, "scale"_a = 0, "pol"_a = 9.)
-
+        .def("fill",
+            [](Event &self, 
+                py::array_t<int> &pid,
+                py::array_t<int> &status,
+                py::array_t<double> &px,
+                py::array_t<double> &py,
+                py::array_t<double> &pz,
+                py::array_t<double> &energy,
+                py::array_t<double> &mass) {
+                fill_event(self, pid, status, px, py, pz, energy, mass);
+            })
```

Comment:
What about col, acol, scale, pol?

Reply:
If I would add them, I would need to initialize another 2 long zero numpy arrays that I don't need.

Reply:
Yes, fair enough. If we ever need them, we can add them.